sdk/tracing/tracing-opentelemetry/src/main/java/com/azure/tracing/opentelemetry/implementation/AmqpTraceUtil.java

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.tracing.opentelemetry.implementation;
+
+import io.opencensus.trace.Status;
+
+public final class AmqpTraceUtil {
```

Comment:
This is a lot of duplication that doesn't belong in this library. I'd add a reference to azure-core-amqp and use those string constants instead.

Reply:
Where did the mapping from ErrorCondition to status codes come from? maybe that can live in azure-core-amqp if they are common. I know there are a few in azure-core-amqp.